### PR TITLE
Updating documentation for the Skywater ReRAM technology.

### PR DESCRIPTION
**Pull request by QuantamHD**
This doc update includes programming instructions, and technology
details not included in the original release.

Signed-off-by: Ethan Mahintorabi <ethanmoon@google.com>
Signed-off-by: Tim Ansell <tansell@google.com>

### CODE REVIEW

**Comment by QuantamHD** [comment]
@mithro @mguthaus This is an in progress doc commit for the ReRAM technology. I wanted to get out the important details as soon as possible. There's still a few seconds on multiple bit and multiple transistor programming that still needs to be added.

File: docs/background.rst [hunk 1]
```diff
@@ -0,0 +1,218 @@
+############
+ Backgorund
```
Comment (mithro) [inline docs/background.rst:2]:
```suggestion
 Background
```

**Review by mithro — COMMENTED** [review]
Change you fix the names of the images like `.-014.png` to be meaningful?

**Review by mithro — APPROVED** [review]
I think the `.-0XX.png` files should be kept separate for now.

Otherwise LGTM.

**Pull request merged**